app/controllers/test_controller.rb

```diff
@@ -118,6 +118,13 @@ def new_task
     render_success 'task', t
   end
 
+  def new_team_data_field()
+    Team.current = Team.find(params[:team_id])
+    team_data_field = "Team-#{params[:fieldset]}"+Time.now.to_s
```

Comment:
Please add spaces around `+`

Reply:
Actually it's better to rewrite the line like this: `team_data_field = "Team-#{params[:fieldset]}-#{Time.now}"` - this way you avoid mixing interpolation and concatenation in the same line.